app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    splitted = command.split(" ")
+    cm = splitted[0]
+    initial_file_name = splitted[1]
+    new_file_name = splitted[2]
```

Comment:
Consider using more descriptive variable names for `cm`, `initial_file_name`, and `new_file_name` to improve code readability. For example, `command_name`, `source_file_name`, and `destination_file_name` would be clearer.